src/main/resources/application.yml

```diff
@@ -11,4 +11,27 @@ spring:
     show-sql: true
     generate-ddl: true
     properties:
-      hibernate.dialect: org.hibernate.dialect.PostgreSQLDialect
+      hibernate.format_sql: true
+  security:
+    user:
+      name: user
+      password: 12345
+    oauth2:
+      resourceserver:
+        jwt:
+          issuer-uri: http://localhost:8081/realms/spring
+logging:
+  level:
+    org:
+      springframework:
+        security:
+          web:
+            FilterChainProxy: Debug
```

Comment:
Nesting is not needed.
I think this way would be better because it's only one value.
`logging.level.org.springframework.security.web.FilterChainProxy: Debug`

Reply:
this is how yaml works, if you do it as `logging.level.org.springframework.security.web.FilterChainProxy: Debug` it will give an error.

Reply:
No. it works :)
Tried it before.

Reply:
I tried it and it indeed works (i stand corrected o.o), but lets keep the format the same across the file.

Reply:
OK, but it's not the same actually :P
You used this "shortcut" just a few lines above in same file :)